Library/Formula/d-bus.rb

```diff
@@ -29,8 +41,6 @@ def install
     system "make"
     ENV.deparallelize
     system "make", "install"
-
-    (prefix/"org.freedesktop.dbus-session.plist").chmod 0644
```

Comment:
Unless users have nonstandard permissions on their homebrew installation directories this is not necessary. The D-Bus make/config scripts will not chmod that file; it appropriately inherits permissions when installed without the chmod.